met-web/src/components/common/index.tsx

```diff
@@ -61,13 +61,13 @@ export const StyledWidgetButton = styled(MuiButton)(() => ({
 }));
 
 export const WidgetButton = ({ children, ...rest }: { children: React.ReactNode; [prop: string]: unknown }) => (
-    <StyledWidgetButton {...rest} variant="outlined">
+    <StyledWidgetButton {...rest} sx={{ minHeight: '40px', maxHeight: '40px' }} variant="outlined">
```

Comment:
let's move these styles to the StyledWidgetButton component above this (line 50)